packages/core/src/internal/decorators/global-style.spec.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { LitElement, html, css } from 'lit-element';
+import { render } from 'lit-html';
+import { registerElementSafely } from '@clr/core/internal';
+import { createTestElement, removeTestElement } from '@clr/core/test/utils';
+import { globalStyle } from './global-style.js';
+
+export class TestElement extends LitElement {
+  static get styles() {
+    return css`
+      :host {
+        --color: red;
+        --background: red;
+      }
+    `;
+  }
+
+  @globalStyle() globalStyles = css`
+    test-global-style-decorator {
+      --color: blue;
+    }
+  `;
+
+  @globalStyle({ static: false }) globalStylesNonStatic = css`
```

Comment:
`@globalStyle` allows a css template to be appended to light dom. Needed for browser specific pseudo selectors. The styles are typically scoped by the element tag.

```css
cds-range ::some-global-style {
}
```